src/v2/cookbook/debugging-in-vscode.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Debugging in VS Code and Chrome
+type: cookbook
+order: 6
+---
+
+Every application reaches a point where it's necessary to understand failures, small to large. In this recipe, we explore a few workflows for VS Code users, who are using Chrome to test.
+
+This recipe shows how to use the [Debugger for Chrome](https://github.com/Microsoft/VSCode-chrome-debug) extension with VS Code to debug Vue.js applications generated by the [Vue CLI](https://github.com/vuejs/vue-cli).
+
+## Prerequisites
+
+You must have Chrome and VS Code installed. Make sure to the latest version of [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome) extension installed in VS Code.
+
+Install and create a project with the [vue-cli](https://github.com/vuejs/vue-cli) by running
+
+```
+npm install -g @vue/cli
+# or
+yarn global add @vue/cli
+
+vue create my-project
+```
+
+Change to the newly created application directory and open VS Code.
+
+```
+cd my-project
+
+# if you have an alias set up (recommended)
+code .
+```
```

Comment:
This part doesn't necessarily have to be done in the terminal and I've found some people prefer to instead open VS Code, then open a folder from there. Maybe we should assume knowledge of how to use the basics of VS Code and just ask people to open the project folder? That way, we're not assuming an alias is set up and that this is how the user prefers to open projects.

Reply:
Yeah, we don't mention here how to set up the alias, either, so this might avoid confusion